reducers/views/threads.ts

```diff
@@ -31,7 +31,24 @@ export const lastViewedAt = (state: ViewsState['threads']['lastViewedAt'] | Reco
     return state;
 };
 
+export function manuallyUnread(state: ViewsState['threads']['manuallyUnread'] | Record<string, unknown> = {}, action: GenericAction) {
```

Comment:
Why is the `Record<string, unknown>` needed here? It seems like that makes the typing information here much less useful. If nothing else, I think the `unknown` could be `boolean`